### ISSUE TITLE
Swatch: automate WCAG color ratio for documentation 

### ISSUE THREAD

**Issue by blackfalcon** [AlaskaAirlines/auroCore-swatch#21]
**Is your feature request related to a problem? Please describe.**

The output of the swatch is a listing of colors and a WCAG rating. The opportunity here is to:

1. Automate this rating
2. Add support for showing the WCAG ratio based off of token data

**Describe the solution you'd like**

Use the WCAG API to return an object that can be used to show the rating and ratio

https://webaim.org/resources/contrastchecker/?fcolor=f8f8f8&bcolor=FFFFFF&api

```
{"ratio":"1.06","AA":"fail","AALarge":"fail","AAA":"fail","AAALarge":"fail"}`
```

The ratio only comes back as the base number, `1.06` so the `:1` would need to be appended manually to illustrate the full ratio, `1.06:1`. 

Additionally, it will be great to have a way to illustrate the different ratings

```
{,"AA":"fail","AALarge":"fail","AAA":"fail","AAALarge":"fail"}`
```

**Comment by Dayna-A** [comment]
IP Task List:
- [ ] add popover with Information about WCAG Contrast Ratios/Ratings
- [ ] add columns to display ratio:1 and ratings
- [ ] retrieve wcag ratings from webaim.org in FirstUpdated() using cacheFetch
- [ ] create icons for normal/Large text AA, AAA or FAIL ratings
- [ ] write UI tests for new ratings & ratios

**Comment by blackfalcon** [comment]
@Dayna-A what is the status of this task? I am confused? Is this work done?

**Comment by Dayna-A** [comment]
@blackfalcon I think I may have reopened this in error.
